src/main/resources/assets/minecolonies/lang/manual_en_us.json

```diff
@@ -49,6 +49,9 @@
   "minecolonies.config.forceloadcolony": "Chunk Load Colony",
   "minecolonies.config.forceloadcolony.comment": "If part of the colony is loaded by an owner/officer, should the colony be kept loaded? (Set how many chunks are loaded with the \"Colony Chunk Loading Strictness\" option.)",
 
+  "minecolonies.config.loadtime": "Colony chunk loading duration",
```

Comment:
(in minutes)

Reply:
well the comment mentions its default 10 mins